authentik/policies/unique_password/apps.py

```diff
@@ -0,0 +1,9 @@
+"""authentik Unique Password policy app config"""
+
+from django.apps import AppConfig
+
+
+class AuthentikPoliciesUniquePasswordConfig(AppConfig):
```

Comment:
```suggestion
from authentik.blueprints.apps import ManagedAppConfig


class AuthentikPoliciesUniquePasswordConfig(ManagedAppConfig):
```

> When I moved the "write password on user_write signal" code, I'm no longer able to get my test_stages.pytests to pass. It seems like user_write is not received inside the policies.unique_password package.

By default, Django doesn't load whatever is in `signals.py`. [This wrapper](https://github.com/goauthentik/authentik/blob/main/authentik/blueprints/apps.py#L49) does it for us, and also loads tasks if they exist, and so on.